app/src/main/java/com/ivanmagda/habito/models/HabitRecord.java

```diff
@@ -42,17 +42,16 @@ public HabitRecord() {
     }
 
     public HabitRecord(String userId, long createdAt, String name, int color, int target,
-                       String resetFreq, long resetTimestamp, int reminderHour, int reminderMin,
-                       int score, List<Long> checkmarks) {
+                       ResetParameterObject resetParam, ReminderParameterObject reminderParam, int score, List<Long> checkmarks) {
```

Comment:
These POJO classes do have some usefulness as the information they contain is more or less coherent.
They can also be extended later and can also receive extra functionality that is only relevant to e.g. ReminderParameterObject and not to the HabitRecord.
So if you don't mind, I would like to keep them as is.